weblate/trans/forms.py

```diff
@@ -1686,10 +1686,27 @@ class Meta:
             "language_regex",
             "source_language",
             "is_glossary",
+            "agreement",
+            "merge_style",
+            "commit_message",
+            "add_message",
+            "delete_message",
+            "merge_message",
+            "addon_message",
+            "pull_message",
         ]
         widgets = {
             "source_language": SortedSelect,
             "language_code_style": SortedSelect,
+            "license": forms.HiddenInput(),
```

Comment:
Why this is hidden input now? This is not supposed to remove choices which were present before.